tests/ignite/metrics/test_entropy.py

```diff
@@ -0,0 +1,206 @@
+import os
+
+import numpy as np
+import pytest
+import torch
+
+import ignite.distributed as idist
+from ignite.exceptions import NotComputableError
+from ignite.metrics import Entropy
+
+
+def np_entropy(np_y_pred: np.ndarray):
+    np_y_pred = np_y_pred - np_y_pred.max(axis=1, keepdims=True)
+    prob = np.exp(np_y_pred) / np.sum(np.exp(np_y_pred), axis=1, keepdims=True)
+    log_prob = np_y_pred - np.log(np.sum(np.exp(np_y_pred), axis=1, keepdims=True))
+    np_ent = -np.sum(prob * log_prob) / np_y_pred.shape[0]
+    return np_ent
```

Comment:
I wonder whether we could use scipy entropy as ref code ?
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.entropy.html
We have scipy for dev dependencies: https://github.com/pytorch/ignite/blob/5fe74432ed5813d803b5c40641cd47d0cb9964df/requirements-dev.txt#L9
so if it is possible to reuse scipy I would prefer to use it instead of manually computing it here.
what do you think @kzkadc ?

Reply:
Thanks for your suggestion!
Indeed, scipy has softmax and entropy.
I updated `np_entropy` to use them.